qualtran/bloqs/arithmetic.py

```diff
@@ -212,3 +219,74 @@ def t_complexity(self):
         # See: https://github.com/quantumlib/cirq-qubitization/issues/219
         # See: https://github.com/quantumlib/cirq-qubitization/issues/217
         return TComplexity(t=8 * self.bitsize)
+
+
+@frozen
+class LessThanEqual(Bloq):
+    r"""Implements $U|x,y,z\rangle = |x, y, x \le y\rangle$.
+
+    Args:
+        x_bitsize: bitsize of x register.
+        y_bitsize: bitsize of y register.
+
+    Registers:
+     - x, y: Registers to compare against eachother.
+     - z: Register to hold result of comparison.
+    """
+
+    x_bitsize: int
+    y_bitsize: int
+
+    @cached_property
+    def signature(self) -> Signature:
+        return Signature(
+            [
+                Register("x", bitsize=self.x_bitsize),
+                Register("y", bitsize=self.y_bitsize),
+                Register("z", bitsize=1),
+            ]
+        )
+
+    def decompose_bloq(self) -> 'CompositeBloq':
+        return decompose_from_cirq_op(self)
+
+    def as_cirq_op(
+        self, qubit_manager: 'cirq.QubitManager', **cirq_quregs: 'CirqQuregT'
+    ) -> Tuple[Union['cirq.Operation', None], Dict[str, 'CirqQuregT']]:
+        less_than = CirqLessThanEqual(x_bitsize=self.x_bitsize, y_bitsize=self.y_bitsize)
+        x = cirq_quregs['x'].tolist()
+        y = cirq_quregs['y'].tolist()
```

Comment:
is the tolist necessary? can't you `*args` unpack numpy arrays as well? please write a small comment if it is indeed required